### PR TITLE
DOC-2439: Export to PDF Standalone Service.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2439

Site: [Staging branch](http://docs-feature-7-doc-2439.staging.tiny.cloud/docs/tinymce/latest/exportpdf/)
Site: [New Page](http://docs-feature-7-doc-2439.staging.tiny.cloud/docs/tinymce/latest/html-to-pdf-converter-api/)

Changes:
* Add `new` export to pdf standalone service page to documentation suite.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/export-to-pdf-standalone-service.adoc [hunk 1]
```diff
@@ -1,5 +1,118 @@
...
+[NOTE]
+Web fonts are not supported in both the header and footer body and styles. For header and footer, use one of the default fonts.
+
+Refer to the link:https://exportpdf.converter.tiny.cloud/docs#section/Web-Fonts[converter documentation^] for details and examples.
+
+== The Export to PDF Editor Plugin
+
+The xref:exportpdf.adoc[Export to PDF] plugin allows easy printing of the WYSIWYG editor content to a PDF file. When enabled, this feature sends the content of the editor together with the styles that are used to display it to the {productname} Cloud Services HTML to PDF converter service. The service then generates a PDF document that can be downloaded.
+
+Thanks to this plugin, it takes exactly one button click to get a PDF file with content formatted the same way as displayed in the {productname} WYSIWYG editor. It provides an easy solution for integrations that do not require further processing of editor data once it is edited.
```
Comment (kemister85) [inline modules/ROOT/pages/export-to-pdf-standalone-service.adoc:110]: @ShiridiGandham I feel like this section could be removed, as its only promoting the actual plugin.

File: modules/ROOT/pages/export-to-pdf-standalone-service.adoc [hunk 2]
```diff
@@ -1,5 +1,118 @@
...
+
+== Stored Data
+
+Export to PDF is a service that generates PDF files based on the data provided by the customer (HTML, CSS). To perform the conversion, it requires the data sent from the editor or a database, but the service itself is fully stateless.
+
+The data, especially sensitive data such as HTML, CSS, comments, is used only during processing. However, some information needed for billing and maintaining the system is stored. The goal is to minimize stored data, currently storing information such as the number and duration of exports, the final document size, the number of unique exports, and statistical data (used features, IP addresses, editor versions).
+
+All logs related to processing requests are saved only in case of unexpected situations and are stored for 14 days. The data in logs include an error message, stack trace, and information about the request.
```
Comment (kemister85) [inline modules/ROOT/pages/export-to-pdf-standalone-service.adoc:118]: @ShiridiGandham Not sure on what is `stored` or captured here if anything is?

File: modules/ROOT/pages/export-to-pdf-standalone-service.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/export-to-pdf-standalone-service.adoc]: removed

File: modules/ROOT/pages/export-to-pdf-standalone-service.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/export-to-pdf-standalone-service.adoc]: removed

File: modules/ROOT/pages/html-to-pdf-converter-api.adoc [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+</html>
+++++
+
+
+== API Reference
+
+> Explore the comprehensive API documentation for the {pluginname} Premium plugin at https://exportpdf.converter.tiny.cloud/docs[API Reference documentation (Export to PDF).^]
+
+== Export to PDF with {productname}
```
Comment (kemister85) [inline modules/ROOT/pages/html-to-pdf-converter-api.adoc:158]:
```suggestion
== Export to PDF Plugin reference
```

File: modules/ROOT/pages/html-to-pdf-converter-api.adoc [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+++++
+
+
+== API Reference
+
+> Explore the comprehensive API documentation for the {pluginname} Premium plugin at https://exportpdf.converter.tiny.cloud/docs[API Reference documentation (Export to PDF).^]
+
+== Export to PDF Plugin reference
+
+> Explore {productname}'s official Export to PDF Premium plugin documentation at https://exportpdf.converter.tiny.cloud/docs[Export to PDF.^]
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/html-to-pdf-converter-api.adoc:160]:
```suggestion
> Explore {productname}'s official Export to PDF Premium plugin documentation at https://www.tiny.cloud/docs/tinymce/latest/exportpdf/[Export to PDF.^]
```

File: modules/ROOT/pages/html-to-pdf-converter-api.adoc [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+</body>
+
+</html>
+++++
+
+
+== API Reference
+
+> Explore the comprehensive API documentation for the {pluginname} Premium plugin at https://exportpdf.converter.tiny.cloud/docs[API Reference documentation (Export to PDF).^]
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/html-to-pdf-converter-api.adoc:156]:
```suggestion
> Explore the comprehensive API documentation at https://exportpdf.converter.tiny.cloud/docs[API Reference documentation.^]
```

File: modules/ROOT/pages/html-to-pdf-converter-api.adoc [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+* **Preprocessing and Customization**: Pre-process content to resolve merge tags and customize PDF document formatting, ensuring that the final output meets specific requirements and is ready for immediate use.
+* **Large-Scale Document Processing**: Utilize the API’s scalability to handle large volumes of PDF documents, maintaining consistent performance and reliability, essential for growing businesses.
+
+== Options Available for HTML to PDF Conversion
+
+When using the converter service, the following options are available:
+
+* link:https://exportpdf.converter.tiny.cloud/docs#section/General/HTML[HTML]
+* link:https://exportpdf.converter.tiny.cloud/docs#section/General/CSS[CSS]
+* link:https://exportpdf.converter.tiny.cloud/docs#section/PDF-options[Options]
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/html-to-pdf-converter-api.adoc:47]:
```suggestion
* link:https://exportpdf.converter.tiny.cloud/docs#section/General/HTML[HTML^]
* link:https://exportpdf.converter.tiny.cloud/docs#section/General/CSS[CSS^]
* link:https://exportpdf.converter.tiny.cloud/docs#section/PDF-options[Options^]
```

**Review by FarzadHayat — APPROVED** [review]
LGTM - good work! 🎉

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**